### DISCUSSION TITLE
Modeling and Simulation

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/sys-581-group-project#36] [General]
Key Questions:

1. How could you utilize Modeling and Simulation to Ensure that your System Works and is Robust?
    - Why?
    - When?
Assessment Criteria - M&S plan is rational and well integrated into the overall system V&V plan.
The objectives/goals of M&S are clearly defined.
2. How could you generate the Simulation inputs?
Assessment Criteria - Simulation inputs and experiments are clearly defined and correlated with the M&S objectives.
4. How could you be sure that you can be confident in
the Simulation results?
Assessment Criteria - Simulation results are clearly correlated with the M&S objectives.

**Comment by dyc3** [comment]
Possible simulations we could do

- Orbital simulations to determine optimal node placements
- Simulate the HTMQ under load
    - Input will be large amounts of video and image data ranging from ~50 GB - ~100GB per message to test how the HTMQ handles varying loads.
- Simulate DSP under load
- Radio simulations to verify that we get our desired range for SRR subsystem

**Comment by juan0campo8** [comment]
Test Optical Data Transmission by modifying noise influences (Signal-to-Noise ratio)
Simulate background light intensity and power fluctuations
Radio Frequency interference testing

**Comment by Victor-M-Giraldo** [comment]
How could you utilize Modeling and Simulation to Ensure that your System Works and is Robust? 

Test the system and the various subsystems under varying amounts of load to push them to their limit. Simulate situations in which we expect the system/subsystems to break?

Why?
Quick successful data transfer is crucial

When?
Before the system is fully deployed.
